docs/usage/commands/interactive.adoc

```diff
@@ -7,6 +7,138 @@ The interactive command will ask several questions about the required projects,
 stacks-cli interactive
 ----
 
+The following table details the questions that will be asked and what they are used for.
+
+.Initial Configuration Questions
+[cols="2,2,1",options=header]
+|===
+| Question | Description | Example
+| 
+What is the name of your company?
+| This is used to help name resources in the chosen cloud platform | Ensono
+
+|
+What is the scope or area of the project?
+| As there can be many different projects being created within a team, this value is designed to be used to categorise the project | core
+
+|
+What component is being worked on?
+| When using Ensono Stacks the scaffolded project is usually being created as part of a larger solution. This value is used to help name the project | backend
+
+|
+What pipeline is being used?
+| A number of different pipelines can be configured to work with Ensono Stacks. Select the pipeline that this project will use. | azdo
+
+|
+Which cloud is Ensono Stacks being setup in?
+| Ensono Stacks supports being deployed into different cloud platforms. Select the cloud that will be used. | azure
+
+| 
+Which group is the Terraform state being saved in?
+| This is the group that contains the storage account for the Terraform state.
+| `stacks-ancillary-resources` 
+
+|
+What is the name of the Terraform storage?
+| The name of the Azure Storage account or S3 bucket being used to store the Terraform state | `lj9sadflkj9`
+
+|
+What is the name of the Terraform storage container?
+| This is the container within the storage that the Terraform state is saved to | tfstate
+
+|
+What is the external domain of the solution?
```

Comment:
Good point. I will put a note on it for the moment
I am going to be redoing the interactive question system so this will be easier. However this is not an easy one to hide because it is asked before the projects are asked so it does not have any context to work with.